config/routes.rb

```diff
@@ -2,9 +2,12 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show] do
+      resources :comments, only: [:create]
```

Comment:
![image](https://user-images.githubusercontent.com/46839548/179376073-da592290-789a-40e6-a82b-af76f91b5be2.png)

- [ ] It is a requirement that the user should be able to create a new post, and this can only happen if the route `/posts/new` is available alongside the `views/posts/new.html.erb` view file. Please create that route 🙏 

**NB** Please note that there should be a form in the views/posts/new.html.erb file which accepts input for creating posts.

- [ ] According to the project requirements, you need to add a `/posts/:post_id/comments/new` route along with a `views/comments/new.html.erb`. Note that there should be a form in the new.html.erb file which accepts input for creating comments.

Reply:
### Hi @codepantha,
Thank you for your feedback I will work on it right now.